app/javascript/file-upload.js

```diff
@@ -124,11 +124,13 @@ MOJFrontend.MultiFileUpload.prototype.getErrorHtml = function(message) {
 
 };
 
-if (typeof MOJFrontend.MultiFileUpload !== "undefined") {
-    new MOJFrontend.MultiFileUpload({
-        container: $(".moj-multi-file-upload"),
-        uploadUrl: window.location.href,
-        deleteUrl: window.location.href
-    })
-}
+// $(function() {
```

Comment:
I _think_ this isn't supposed to be commented out, but let me know if there's some clever JS magic going on that's gone above my head.

Reply:
i was testing was an actual fix by turning it off... opps...